### ISSUE TITLE
Create contraptions generate broken block fragments when activated

### ISSUE THREAD

**Issue by LuckyChickens** [haubna/PhysicsMod#699]
I use the Physics mod alongside the Create mod, and I have noticed that whenever a contraption starts moving (it transforms from a bunch of blocks into a moving entity), it spawns a bunch of fragments as if those blocks had been broken regularly. This also occurs with the Carry On mod when picking up a block entity such as a chest.
Is there some addon or setting that I don't know about to prevent this? Or will this need a future update to fix?

**Comment by MalevolentWulf** [comment]
Do you perhaps have dynamic blocks on? If so try to toggle that off and see if it helps

**Comment by LuckyChickens** [comment]
I don't have the pro version, so I don't think dynamic blocks should be the issue.

**Comment by Deep-Shift** [comment]
btw the pro version is free just go to haubna's patron and scroll find the most recent update post and click it scroll a bit and you'll see a bunch of file name find your version and mod loader and click it to download here 1.19.2 forge and fabric download links Fabric: [https://www.patreon.com/file?h=86127166&i=14977350](url) Forge: [https://www.patreon.com/file?h=86127166&i=14977354](url)

**Comment by LuckyChickens** [comment]
So I installed physics mod pro, and I noticed something interesting.

First, with dynamic blocks turned on, blocks such as chassis and mechanical bearings from create do not generate physics voxels when broken. Normal blocks like dirt still retain their behavior.

Second, multiblock create contraptions do not generate physics voxels when activated. Contraptions made up of just a single block will still act like they were broken by hand, however.

Whenever dynamic blocks is turned off, everything makes voxels as if they were broken by hand.

So the solution appears to be installing the pro version and making sure dynamic blocks is on.
